packages/components/src/components/AlertDialog/AlertDialog.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { Dialog } from "ariakit/dialog";
+import type { DialogProps } from "ariakit";
+import { Box } from "../../primitives/Box";
+
+export interface AlertDialogProps extends Omit<DialogProps, "noonce"> {
+  /** The contents of the alert dialog. */
+  children: NonNullable<React.ReactNode>;
+  /** Centers the content of the alert dialog. */
+  centered?: boolean;
+}
+
+/** A dialog component that forces the user to acknowledge and make a choice */
+const AlertDialog = React.forwardRef<HTMLDivElement, AlertDialogProps>(
```

Comment:
Also we should be able to add a role here of `alertdialog`. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alertdialog_role#:~:text=The%20alertdialog%20role%20is%20used,rest%20of%20the%20page%20content.